Xamarin.Forms.Platform.Android/Renderers/TimePickerRenderer.cs

```diff
@@ -109,6 +115,25 @@ protected virtual TimePickerDialog CreateTimePickerDialog(int hours, int minutes
 			return dialog;
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (_isDisposed)
+				return;
+
+			_isDisposed = true;
+
+			if (disposing)
+			{
+				if (Forms.IsLollipopOrNewer && _dialog != null)
```

Comment:
```suggestion
				if (Forms.IsLollipopOrNewer && _dialog.IsAlive())
```

When we dispose we're going to need to start checking if the object is still alive before we call into native apis. 
This is really important for the previewer but also a good practice to start adopting in general.